### DISCUSSION TITLE
How to modify syntax themes

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#1527] [Help] [answered]
The documentation says it is possible to update syntax themes but I have not been able to find anywhere to explain how this is done. 
I want to use the "light owl" theme, but text between single quotes is changed to yellow, which is impossible to see against the pale background of the theme. How do I change it?
Thanks.

**Comment by vkbo** [comment]
It is explained in the documentation under "Customisations": https://novelwriter.io/docs/int_customise.html#syntax-and-gui-themes

I can also look into improving the contrast of the light owl theme.

**Reply by vkbo** [reply]
Reply by @mogplus88 moved here from a separate answer:

> Okay, created a new theme and copied it to my ~/.local/share/novelwriter/themes but it doesn't come up in the dropdown in the preferences dialog. I've tried logging out and in again, and rebooting, but it still doesn't appear. Not sure what I've missed...

**Reply by vkbo** [reply]
Please reply to the answer instead of replying as a "new answer" to your own question. See #1393.

The difference between `syntax` and `theme` is explained in the link I provided. Adding a new file in `theme` will not show up under `syntax`. Also, make sure you give it a new name.

**Reply by mogplus88** [reply, marked as answer]
Sorry, typo. I copied the light owl theme and changed the setting for singlequotes to 40, 142, 215 (blue) and changed the name to "My Theme" and saved it in the syntax folder under novelwriter. The file name was MySyntaxTheme, which I've now figured out was the problem. It should be MySyntaxTheme.conf. When I changed that, the theme appeared as expected. 
Apologies for wasting your time.
:-\

**Reply by vkbo** [reply]
Oh, don't worry about it. Glad it worked 🙂
